backend/src/services/weather.ts

```diff
@@ -0,0 +1,16 @@
+export const getCities = async () => {
```

Comment:
When will this be done? Happy to use a static file in FE for now.

Reply:
I'll confirm with Julian, it might be something he wants to extend or not. The API that provides you with a list of cities are up and running under routes  /cities